BerlinDE.ts

```diff
@@ -0,0 +1,30 @@
+import axios, { AxiosResponse } from 'axios';
+import cheerio from 'cheerio';
+
+const BASE_URL = 'https://www.berlin.de';
+const NEWS_PATH = '/en/news/';
+
+export async function fetchArticles(): Promise<Array<BerlinDEArticle>> {
+  const response: AxiosResponse<string> = await axios(
```

Comment:
You could use the method `get` which already has an `AxiosResponse` and that receives a generic, see here: https://github.com/axios/axios/blob/master/index.d.ts#L140

Reply:
Do you mean something like:
```javascript
const response: AxiosResponse<string> = await axios.get<string>(
  `${BASE_URL}${NEWS_PATH}`,
);
```

Reply:
?

Reply:
Yes, then `AxiosResponse<string>` is no longer necessary, once `axios.get` returns is type-safed.

Reply:
Ah, I'm just making it type explicit. I think I got your point now 😄

Thanks =]